experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -912,7 +912,7 @@ describe("PageSummary", () => {
       targetingConfigSlug: "OH_NO",
     });
     render(<Subject mocks={[mock]} />);
-    expect(screen.queryByTestId("bucketing-warning")).toBeInTheDocument();
+    expect(screen.queryByTestId("warning-bucketing")).toBeInTheDocument();
```

Comment:
nit: replace the `expect(...).toBeInTheDocument()` with `screen.getByTestId(...)`

`getByTestId` will throw if it does not find the selector.

Reply:
We should definitely file some cleanup tickets, because `expect(...).toBeInTheDocument()` is all over in our tests 😖 I'll fix these for now since this is a refactor PR, but it would be great to give them all some love